### PR TITLE
Update README.md: Add description about main branch, delete unnecessary contents

### PR DESCRIPTION
**Pull request by chenrujun**
## Purpose

* Now the main branch only uses the latest stable version of **Spring Cloud Azure** libraries. We should make this information clearer in the README.md.
* How to create Azure resources already exists in each sample's README.md. So, it can be deleted in the root README.md.
* Since we will list sample's name and description in https://github.com/Azure-Samples/azure-spring-boot-samples/pull/597. Spring Native is just a sample. It will appear in the samples table. No need to put it in a standalone section. So, I deleted it in this PR.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```
may be needed here. -->

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @fangjian0423 please help to review this PR when you have time.

File: README.md [hunk 1]
```diff
@@ -1,37 +1,9 @@
...
 
-# Spring Cloud Azure Samples
-- This is a sample project for demonstration purposes.   
-- The project demonstrates how to use [Spring Cloud Azure](https://microsoft.github.io/spring-cloud-azure/current/reference/html/index.html) to develop applications.    
-- The sample project contains 42 samples, and it still grows.    
-- These samples are grouped by Azure services and [Spring Cloud Azure libraries](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/spring).    
+# Spring Cloud Azure Samples  
+- The repository hold samples about using [Spring Cloud Azure](https://learn.microsoft.com/azure/developer/java/spring-framework/) (**SCA**) libraries.
```
Comment (fangjian0423) [inline README.md:4]: Don't use SCA, we discuss it in today's scrum. keep using Spring Cloud Azure

File: README.md [hunk 2]
```diff
@@ -1,37 +1,9 @@
...
-    >
-    >  **servicebus**: The Azure service that the sample integrated with.
-    >
-    > **spring-cloud-azure-starter-integration-servicebus**: The dependency that the sample depends on.
+# Getting Help
+- If you have any question about using these samples, please [create an new issue](https://github.com/Azure-Samples/azure-spring-boot-samples/issues/new/choose).
 
-- The main branch is using the latest stable version of Spring Cloud Azure libraries. If you want to use another version, please checkout tag according to your required version.
-
-## Running Samples With Terraform
```
Comment (fangjian0423) [inline README.md:17]: Why delete terraform section? I think it is useful.

File: README.md (hunk 1 above)
Comment (chenrujun) [reply README.md:4]: Will update this.

File: README.md (hunk 2 above)
Comment (chenrujun) [reply README.md:17]: Each sample's README already has content about using terraform to create required resources. It's redundant to add this information in the root README.

**Review by fangjian0423 — APPROVED** [review]
LGTM

**Pull request merged**